applicant-frontend/src/static.json

```
@@ -1,13 +1,13 @@
 {
-  "ERROR_REQUIRED_QUESTIONS": "Please answer correctly to required questions",
+  "ERROR_REQUIRED_QUESTIONS": "Please answer correctly to required questions.",
   "ERROR_VALID_PASSWORD": "Please provide a valid password. (10 characters: at least one uppercase, lowercase and one number)",
-  "ERROR_VALID_PHONE": "Please provide a valid phone number (for example +77123123123) no space",
-  "ERROR_VALID_EMAIL": "Please provide a valid email address",
-  "SUBMIT_FORM_VALIDATION": "Are you sure to submit this form? If you prefer to edit it later, save it.",
+  "ERROR_VALID_PHONE": "Please provide a valid phone number (for example 6934303600) no space",
```

```suggestion
  "ERROR_VALID_PHONE": "Please provide a valid phone number (for example +6934303600) no space",
```
We need the '+' in the answer

---

I am not sure, they will get it the +30. They might not even knoe the country code. In the worst case scenarion, they will copy paste the suggestion as a number. But you mean that the platform cannot work without the + ???

---

So, in the code you wrote, which version of phone number is acceptable?